src/clients/AcrossConfigStoreClient.ts

```diff
@@ -208,9 +209,12 @@ export class AcrossConfigStoreClient {
   }
 
   getConfigStoreVersionForTimestamp(timestamp: number = Number.MAX_SAFE_INTEGER): number {
-    const config = (sortEventsDescending(this.cumulativeConfigStoreVersionUpdates) as ConfigStoreVersionUpdate[]).find(
-      (config) => config.timestamp <= timestamp
-    );
+    const config = this.cumulativeConfigStoreVersionUpdates.find((config) => config.timestamp <= timestamp);
+    return isDefined(config) ? Number(config.value) : DEFAULT_CONFIG_STORE_VERSION;
+  }
+
+  getConfigStoreVersionForBlock(blockNumber: number): number {
```

Comment:
Why would you need these two functions? IIUC you would need to use this function to figure out which config store parameters to apply to a deposit/fill. These events all have `quoteTImestamps` attached to them so you should always be able to compare by timestamp, which is how the client is currently set up. When would you ever need to use block? And in those cases, wouldn't you always be able to grab the timestamp for that block?

Reply:
Great point...have left that as a comment here: https://github.com/across-protocol/sdk-v2/pull/213/files#r1223234563.